package.json

```diff
@@ -77,7 +76,6 @@
     }
   },
   "volta": {
-    "node": "16.14.0",
-    "npm": "7.24.2"
+    "node": "18.17.1"
```

Comment:
**(question)**: do we need to specify the npm version in volta as well?

Reply:
[Comment] I don't think we need the NPM version to be specify anymore.

Reply:
**(question)**: oh do you know why that is?

Reply:
[Comment] I don't know why but the ones i have migrated doesn't include the NPM version and i suppose toolkit is no longer requiring that

Reply:
**(question):** could we make sure this is deliberate and not something that was overlooked by the platforms team?